src/config/cfg_rules.ini

```diff
@@ -578,13 +578,13 @@ option = krb5_ccname_template
 option = krb5_confd_path
 option = krb5_fast_principal
 option = krb5_fast_use_anonymous_pkinit
+option = krb5_kdcinfo_lookahead
```

Comment:
I would place it near `krb5_use_kdcinfo` but doesn't matter much, of course.

Reply:
The options here are sorted alphabetically.

Reply:
Ah, ok.